tools/reviewer/web/src/styles/singleReview.scss

```diff
@@ -3,6 +3,17 @@
 .single-review {
   background-color: #f3f3f3;
 
+  .actions {
```

Comment:
That's one way of looking at it but a lot of times components have a lot of child components and managing css in so many files becomes difficult. for example Review page has about 4, 5 components now that means css code split among 5 files. It would get difficult to manage that. True power of scss is nesting is wasted if there are too many scss files.

Reply:
It's not more difficult to write scss code separately. You always able to import variables and reuse scss files if necessary.
But one huge scss file confuses. And it becomes harder and harder to maintain such files.
Also, the file even not located in review folder. When other developers will see your code, they don't know where they should find scss code of a component.

This is the same issue as with the huge components. I'm glad that you separate them. Maybe it's time to separate scss files too?

Reply:
Yeah, we can use variables etc but we can't use nesting and styles are separated in around 5-6 files. It'll be more difficult to make sense of it. As for finding it, it's name is pretty logical there's a file reviewsPanel.scss and login.scss and singleReview.scss. It utilizes nesting and that helps with using less class names. It's easier to find all the styles when they are together and don't have to open up  a lot of files.

It's just different way of looking at stuff just as you like a different folder structure rather than the default Angular folder structure.

Reply:
It's not a way of looking at stuff. It's conglomeration a lot of code in a single file. A file, which is located in an unexpected place. When developers look at a component, they expect to find styles somewhere near the component. Or, at least, to have a link to styles in the component. But your way of thinking doesn't provide that.

And what nesting are you talking about?
If a component and its children have an element with the same style, e.g. a button the fixed font and color, you could just create a scss file with styles related to the button only and put the file to all components.
Do you know, that a component can have more that one scss file?
If some element has the same parament, e.g. div with a fixed width, you could import width variable to all scss, where the element is.
That's it. Maybe it's sound a little bit more difficult, than to drop all code in a single file, but that keeps code clean and much more readable for you and other developers.